<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TxGeneratorWindow</class>
 <widget class="QWidget" name="TxGeneratorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Generator</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutMain">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTop">
     <item>
      <widget class="QLabel" name="labelInterface">
       <property name="text">
        <string>Interface:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxInterface">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_Top">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="groupBoxAvailable">
      <property name="title">
       <string>Available Messages (DBC)</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutAvailable">
       <item>
        <widget class="QTabWidget" name="tabWidgetAvailable">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabMessageList">
          <attribute name="title">
           <string>Message List</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayoutTabList">
           <item>
            <widget class="QLineEdit" name="lineEditSearchAvailable">
             <property name="placeholderText">
              <string>Message Name or ID...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeAvailable">
             <column>
              <property name="text">
               <string>ID</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabLayoutView">
          <attribute name="title">
           <string>Layout View</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayoutLayoutRoot">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayoutLayoutControls">
             <item>
              <widget class="QLabel" name="labelZoom">
               <property name="text">
                <string>Zoom:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSlider" name="sliderLayoutZoom">
               <property name="minimum">
                <number>50</number>
               </property>
               <property name="maximum">
                <number>200</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="cbLayoutCompact">
               <property name="text">
                <string>Compact Mode</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_Layout">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QScrollArea" name="scrollAreaLayout">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <layout class="QVBoxLayout" name="verticalLayoutTabLayout">
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutAvailableControls">
         <item>
          <widget class="QPushButton" name="btnAddToList">
           <property name="text">
            <string>Add Selected DBC Message</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelManualId">
           <property name="text">
            <string>ID (Hex):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineManualId">
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask">
            <string>HHHHHHHH</string>
           </property>
           <property name="text">
            <string>000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labelManualDlc">
           <property name="text">
            <string>DLC:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinManualDlc">
           <property name="maximum">
            <number>64</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnAddManual">
           <property name="text">
            <string>Add Manual</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBoxActive">
      <property name="title">
       <string>Active Cyclic Transmissions</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutActive">
       <item>
        <widget class="QTreeWidget" name="treeActive">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <column>
          <property name="text">
           <string>Status</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Interface</string>
          </property>
         </column>
          <column>
           <property name="text">
            <string>DLC</string>
           </property>
          </column>
         <column>
          <property name="text">
           <string>Interval (ms)</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutActiveControls">
          <item>
           <widget class="QPushButton" name="btnBulkRun">
            <property name="text">
             <string>RUN</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnBulkStop">
            <property name="text">
             <string>STOP</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         <item>
          <widget class="QLabel" name="labelInterval">
           <property name="text">
            <string>Interval:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinInterval">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100000</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendOnce">
           <property name="text">
            <string>Send Once</string>
           </property>
          </widget>
         </item>
          <item>
           <widget class="QPushButton" name="btnSelectAll">
            <property name="text">
             <string>Select All Rows</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearAll">
            <property name="text">
             <string>Deselect All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRemove">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
